tests/foreman/cli/test_satellitesync.py

```diff
@@ -1160,7 +1177,7 @@ def test_negative_export_cv_with_puppet_repo(self):
         )
 
     @tier3
-    def test_negative_export_cv_with_mixed_content_repos(self):
+    def test_postive_export_cv_with_mixed_content_repos(self):
         """Exporting CV version having yum and non-yum(puppet) repos throws error
```

Comment:
If the intention of the test has changed from negative to positive, I'm assuming you will not be required say 'throws error' ? Please update the docstring with valid intent.